src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -219,6 +226,7 @@ const getAvailablePaymentMethods = (
   if (availablePaymentMethods.includes("WIRE_TRANSFER")) {
     paymentMethods.push(
       <BorderedRadio
+        key={2}
```

Comment:
I know it doesn't matter, but would it make sense to make these keys simply the paymentMethod?

Reply:
yes it's actually better to use strings as keys!